.github/workflows/security.yaml

```diff
@@ -7,7 +7,8 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: '^1.19.1'
+        go-version: '^1.19'
+        check-latest: true
```

Comment:
Yes, using `check-latest` will cause it to make a call out to [go-versions](https://github.com/actions/go-versions) to see if it's the latest version, but that's a quick call and once it's in cache, it'll be fairly performant anyway:
https://github.com/actions/setup-go/blob/v3.3.0/src/installer.ts#L40

This will allow us to just re-trigger off the build in the event there's a security concern, instead of needing to find a way to bust the action's cache, alter the go-version, etc.